### ISSUE TITLE
Google Sheets to GitHub .csv Script Workflow Transfer from Theo to Crystal to Cypress

### ISSUE THREAD

**Issue by CECSpecialistI** [crystalyragui/MARC2RDA#449]
This workflow belonged to Theo, and he explained it very briefly to Crystal who never picked it up. We should run it again and Crystal doesn't know what she's doing. It's Python and Theo described it as having room for improvement and "Cypress could do it in her sleep." This should belong to Cypress, and if she's willing, it would be fabulous if Cypress would write some better documentation for it. Crystal doesn't know where it is. We should probably sit down and meet about it. We'll find where Theo put it. Maybe.

**Comment by cspayne** [comment]
looks like it's [zippedExcel2csv.py](https://github.com/uwlib-cams/MARC2RDA/blob/main/Working%20Documents/Non-Mapping%20Documents/zippedExcel2csv.py)

**Comment by CECSpecialistI** [comment]
That's the one! You found it!

**Comment by cspayne** [comment]
@CECSpecialistI is there a time you wanted to run this again? I can go ahead and work on it if the Google Sheets are ready.

**Comment by CECSpecialistI** [comment]
> @CECSpecialistI is there a time you wanted to run this again? I can go ahead and work on it if the Google Sheets are ready.

No specific time, now would be great. We should probably run it on a regular basis. Maybe monthly?

**Comment by cspayne** [comment]
@CECSpecialistI just ran the code and pushed any updated csv files, you should have been tagged in the commit. The code works great, but I'm going to do a bit of clean up on it and get a README file or wiki page set up on how to run it.

**Comment by CECSpecialistI** [comment]
Thank you for your efforts! All of those things will be extremely useful and appreciated.

**Comment by cspayne** [comment]
@CECSpecialistI 
Everything's been updated and documentation is available in the [Non-Mapping Documents folder](https://github.com/uwlib-cams/MARC2RDA/tree/main/Working%20Documents/Non-Mapping%20Documents). I'm going to go ahead and close this issue.

**Comment by tmqdeborah** [comment]
Two things about the csv file for the [list of MARC21 Bibliographic fields and subfields excluded from the project’s mapping](https://github.com/crystalyragui/MARC2RDA/blob/main/Phase%20I%20Deliverables/Not%20Mapped%20-%20Sheet1.csv)
1. A little earlier when I followed the link, the table did not appear to have come over completely
2. Now when I follow the link, I get a 404 message

**Comment by cspayne** [comment]
Do you mean this file: https://github.com/crystalyragui/MARC2RDA/blob/main/Phase%20I%20Deliverables/Not%20Mapped.csv?

**Comment by tmqdeborah** [comment]
Yes, that link takes me to what seems to be a table that lacks columns. But maybe that is how it is intended to look?

The link from the landing page is: https://github.com/crystalyragui/MARC2RDA/blob/main/Phase%20I%20Deliverables/Not%20Mapped%20-%20Sheet1.csv and gets the 404 message.

The link from https://github.com/crystalyragui/MARC2RDA/tree/main/Phase%20I%20Deliverables, was getting that message, but is now showing the table as it appears when I follow your link. 

I have pasted your link into the [Deliverables Description ](https://marc2rda.info/wiki/doku.php?id=start#deliverables_description)list, and it still come up "not found". Maybe you might have more luck fixing it ?

**Comment by cspayne** [comment]
The table is correct, it just looks funny in GitHub because it doesn't have many columns. I just updated the links and they are working.
